cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/MultiLiveStreamFileReaderTestBase.java

```diff
@@ -46,11 +48,12 @@
   @Test
   public void testMultiFileReader() throws Exception {
     String streamName = "multiReader";
+    Id.Stream streamId = Id.Stream.from(Constants.DEFAULT_NAMESPACE, streamName);
```

Comment:
This PR doesn't support multiple namespaces for streams yet, so its not quite possible.
I will be adding test cases for other namspaces in the PR that actually supports multiple namespaces for streams.